core/src/main/java/org/mapfish/print/map/geotools/AbstractGeotoolsLayer.java

```diff
@@ -116,7 +119,11 @@ public final void render(final Graphics2D graphics2D,
             final ReferencedEnvelope mapArea = bounds.toReferencedEnvelope(paintArea, transformer.getDPI());
             renderer.paint(graphics2D, paintArea, mapArea);
         } catch (Exception e) {
-            throw ExceptionUtils.getRuntimeException(e);
+            if (e instanceof NullPointerException || e.getCause() instanceof NullPointerException) {
+                LOGGER.info("Invalid GeoJSON");
```

Comment:
`AbstractGeotoolsLayer` is not specific to GeoJSON, so the error message "Invalid GeoJSON" might be wrong if a `NullPointerException` is thrown for other layer types. Could you change it to a more generic error message (like "Rendering layer failed") and please also log the exception?